lib/maxsub.gi

```diff
@@ -595,10 +569,41 @@ local id,m,epi,cnt,h;
     fi;
 
   fi;
+  TryNextMethod();
+end);
 
-  if ValueOption("cheap")=true then return [];fi;
-  Info(InfoLattice,1,"MaxesAlmostSimple: Fallback to lattice");
-  return MaxesByLattice(G);
+InstallMethod(MaxesAlmostSimple,"permutation group",true,[IsPermGroup],0,
```

Comment:
[Question] Do we have any guidance regarding writing the free text comment? Is "permutation group" ok here, of "for permutation group" would be better?